dev/build.image/build.gradle

```diff
@@ -559,6 +561,44 @@ if (isAutomatedBuild && !isIFIXBuild) {
         }
     }
     
+    //GA
```

Comment:
GA?!

Reply:
Struggling to remember the details but I have a nagging doubt that creating these zips now can result in them being made public. I need to try to remember why that was.

Reply:
Yeah we need to inform the website guys and Phil not to include them until we tell them to.  I originally wanted to put NOSHIP instead of GA, but looked at what you did previously for EE 10 and did the same and put it as GA.